packages/components/src/Button.tsx

```diff
@@ -220,6 +226,21 @@ Button.propTypes = {
     }
     return null;
   },
+  onMouseUp(props) {
+    const { onMouseUp } = props;
+    if (onMouseUp !== undefined && typeof onMouseUp !== 'function') {
+      return new Error('onMouseUp must be a function');
+    }
+    return null;
+  },
+  onMouseDown(props) {
+    const { onMouseDown } = props;
+    if (onMouseDown !== undefined && typeof onMouseDown !== 'function') {
+      return new Error('onMouseDown must be a function');
+    }
+    return null;
+  },
```

Comment:
You can just write this as `PropTypes.func`. The only reason it's split up in the `onClick` is because it's only checking if the `onClick` function is there for type of `'button'`.

```suggestion
  onMouseUp: PropTypes.func,
  onMouseDown: PropTypes.func,
```